Apps/Admin/Client/Pages/SupportPage.razor.cs

```diff
@@ -173,25 +189,28 @@ private async Task SearchAsync()
                         QueryType = this.SelectedQueryType,
                         QueryString = StringManipulator.StripWhitespace(this.QueryParameter),
                     });
+
+                await this.JsRuntime.InvokeVoidAsync("sessionStorage.setItem", "queryType", this.SelectedQueryType);
+                await this.JsRuntime.InvokeVoidAsync("sessionStorage.setItem", "queryParameter", this.QueryParameter);
```

Comment:
I'd suggest adding a prefix to these keys ("supportQueryType" and "supportQueryParameter") to improve clarity and prevent clashes in case we want to implement something similar on other pages.

Reply:
Fixed.